projects/vdk-heartbeat/src/vdk/internal/heartbeat/config.py

```diff
@@ -21,8 +22,12 @@ def __init__(self, config_file=None):
             log.info(f"Configuration loaded: {self._config_ini.items()}")
 
         # VDK CLI Arguments for vdkcli login
-        self.vdkcli_api_refresh_token = self.get_value("VDKCLI_OAUTH2_REFRESH_TOKEN")
-        self.vdkcli_oauth2_uri = self.get_value("VDKCLI_OAUTH2_URI")
+        self.vdkcli_api_refresh_token = self._get_atleast_one_value(
+            "VDKCLI_OAUTH2_REFRESH_TOKEN", "VDK_OAUTH2_REFRESH_TOKEN"
```

Comment:
Let's name the new variables VDK_HEARTBEAT_API_TOKEN - That's the official name. 

And let's prefix all vdk heartbeat (only) specific options with "VDK_HEARTBEAT_" 

Refresh token is an old name that was not changed in vdk-heartebat. (see vdk login --help) .

Reply:
Renamed, thanks